src/components/Shared/Map/CrossingMap.js

```diff
@@ -164,7 +164,6 @@ class CrossingMap extends React.Component {
     if (map) {
       map.flyTo({
         center: point,
-        zoom: 15,
       });
```

Comment:
This is a great thing for when we're tapping on the map because random zooms get confusing, however, we set this zoom on flyto based on the behavior we wanted when someone selects a crossing from the search.

There are currently three ways to select a crossing:
* Click/tap the map
* Use the search bar
* Use the nearby crossings

In each scenario we should understand from a design stand point what behavior we're looking for.
We have a few options here:
* Never zoom, always show tooltip
* Zoom on search only, don't zoom on nearby or map tap
* Other?

Reply:
made issue #299